app/lint.xml

```diff
@@ -13,5 +13,7 @@
         <ignore path="**/ic_person_24dp.png"/>
         <ignore path="**/ic_search_24dp.png"/>
     </issue>
-
+    <issue id="UnusedNamespace" severity="error"/>
```

Comment:
These all can be added as well 
OnClick
MissingPermission
UnknownId
DuplicateIds
SdCardPath
NewerVersionAvailable
RequiredSize
Typos


Warnings:
PluralsCandidate
StaticFieldLeak